Shared/messages/MessagesOverlay.h

```diff
@@ -18,6 +18,8 @@
 #define MESSAGESOVERLAY_H
 
 // Qt headers
+#include <QHash>
+#include <QString>
```

Comment:
Can you forward declare QString?

Reply:
QString is not necessary since it's already included by QObject. Removed